daily-emailing/daily_email.py

```diff
@@ -45,7 +46,7 @@ def send_email() -> None:
     print("sent email")
 
 
-def lambda_handler(event: dict, context: dict) -> dict:
+def lambda_handler(event: dict, context: dict) -> dict:  # pylint: disable=W0613
```

Comment:
Good to disable this pylint error since there's not always an event or context defined locally.